src/modules/image/Image.cpp

```diff
@@ -0,0 +1,26 @@
+#include "faker-cxx/Image.h"
+
+#include "faker-cxx/Number.h"
+#include "fmt/format.h"
+
+namespace faker
+{
+
+std::string Image::imageUrl(unsigned int width, unsigned int height, std::optional<ImageCategory> category)
+{
+    const std::string image_category =
+        category.has_value() ? fmt::format("/{}", imageCategoryString.at(category.value())) : "";
+    return fmt::format("https://loremflickr.com/{}/{}{}", width, height, image_category);
+}
+
+std::string Image::githubAvatarUrl()
+{
+    return fmt::format("https://avatars.githubusercontent.com/u/{}", Number::integer<int>(100000000));
+}
+
+std::string Image::dimensions()
+{
+    return fmt::format("{}x{}", Number::integer<int>(1, 32720), Number::integer<int>(1, 17280));
```

Comment:
I have question here. Are we sticking to realistic dimensions? I assume dimension like "0x100" is invalid so we don't want it. And also about max for now this is 32k as max. I can increase if we want bigger

Reply:
Yes, we want realistic values, those ranges are fine :)